release_notes/unreleased.md

```diff
@@ -1 +1,7 @@
 **Unreleased**
+
+* Updated certificate actions to work with API v2
+    * Changed pagination behavior to use 
+    * Created separate module for pagination searches, censys_search.py
+* Code refactorisation, moved helper validation to censys_validation.py 
```

Comment:
This s an internal change and IMHO should not be in a changelog as, if done correctly, is invisible for the user

Reply:
Thanks for clarifying, I'll write them from the user perspective